blocks/shared-styles/_children/promo-date/index.test.jsx

```diff
@@ -15,7 +15,7 @@ jest.mock('fusion:properties', () => jest.fn(() => (
 )));
 
 jest.mock('@wpmedia/engine-theme-sdk', () => ({
-  localizeDateTime: jest.fn(() => new Date().toDateString()),
+  localizeDateTime: jest.fn((x) => new Date(x).toDateString()),
```

Comment:
ew, I'm not sure how I feel about this mock. good job improving it though. I might prefer just to pass through a fake date?